nautobot_secrets_providers/providers/hashicorp.py

```diff
@@ -61,26 +68,33 @@ class ParametersForm(BootstrapMixin, forms.Form):
         )
         mount_point = forms.CharField(
             required=False,
-            help_text=f"The path where the secret engine was mounted on (Default: <code>{DEFAULT_MOUNT_POINT}</code>)",
-            initial=DEFAULT_MOUNT_POINT,
```

Comment:
Yes, there isn't really any way to make this Dynamic, as you would choose your configuration after the form is already loaded. I could add some Javascript to dynamically fetch, but I figured that would be unnecessary, and make this a much larger change.

Reply:
Possibly just remove the `(Default: ...)` text since it's inaccurate/misleading now? So for example this help_text would change to something like "The path where the secret engine is mounted, if different from the default for the selected vault"?